code/modules/hydroponics/plant_genes.dm

```diff
@@ -394,6 +394,10 @@
 	if(!(G.resistance_flags & FIRE_PROOF))
 		G.resistance_flags |= FIRE_PROOF
 
+/datum/plant_gene/trait/settled_traits
+	//The traits cannot be changed on this plant. Would make for a pretty funny traitor item, because you could give this but not take it away.
+	name = "Settled Traits"
```

Comment:
Will this appear randomly in strange seeds?

Reply:
i should fix that